src/shogun/mathematics/graph/Graph.cpp

```diff
@@ -138,8 +140,9 @@ void Graph::add_operator_node(const std::shared_ptr<Node>& node)
 	case GRAPH::XLA:
 	case GRAPH::TVM:
 	case GRAPH::SHOGUN:
-		break;
+		op = create_operator<OperatorShogunBackend>(std::string(node->get_operator_name()));
 	}
+	op->build(node);
```

Comment:
@vigsterkr this adds the link between the "abstract representation" of the graph with the implementation